html/mycakeapp/src/Controller/MoviesInfoController.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+namespace App\Controller;
+
+use App\Controller\AppController;
+
+
+use Cake\Event\Event; // added.
+use Exception; // added.
+
+// 今のところappcontrollerを継承
+class MoviesInfoController extends AppController
+{
+    // 初期化処理
+    public function initialize()
+    {
+        parent::initialize();
+        $this->loadComponent('Paginator');
+        $this->loadModel('Movies');
+        $this->loadModel('MovieSchedules');
+
+        // レイアウトをmainに変更
+        $this->viewBuilder()->setLayout('main');
+    }
+
+    public function index()
+    {
+        // ================ 日付について start ================
+        $today = date("Ymd H:i:s");
```

Comment:
[提案]
今回必要なのは日付の`Y-m-d`のみなので、`H:i:s`を削ってもいいかなと思います！
それで42行目を削除して、48・49行目に`$today`を使い回すと同じようなコードを減らせるのではないでしょうか？？
```suggestion
        $today = date("Y-m-d");
```

Reply:
織田さんのアドバイスをヒントにそもそも変数に入れる工程を消しました！
アドバイスありがとうございます。